VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/interactor/LogInScreenInteractor.java

```diff
@@ -24,7 +24,6 @@
     public LogInScreenInteractor observeLogInScreen() {
         final MainApplication app = MainApplication.instance();
         final CharSequence title = ResourceUtil.getFormattedString(app.getResources(), R.string.login_title, "platform_name", app.getInjector().getInstance(Config.class).getPlatformName());
-        onView(withId(R.id.activity_title)).check(matches(withText(title.toString())));
```

Comment:
Instead of removing this assertion, you can update it to check the titlebar text. Here is one way to do that, taken from `MyCoursesScreenInteractor`:
```java
onView(allOf(isInActionBar(), withText(title.toString()))).check(matches(isCompletelyDisplayed()));
```